nvflare/private/fed/utils/fed_utils.py

```diff
@@ -41,6 +41,7 @@
 from nvflare.security.logging import secure_format_exception
 from nvflare.security.security import EmptyAuthorizer, FLAuthorizer
 
+from ..simulator.simulator_const import SimulatorConstants
 from .app_authz import AppAuthzService
```

Comment:
Suggest use absolute imports, but not necessary for this PR

Reply:
we can change it in a later PR.